vendor/github.com/moolitayer/awx-client-go/awx/job_template_launch_resource.go

```diff
@@ -20,6 +20,8 @@ limitations under the License.
 package awx
```

Comment:
Note that the AWX client has bee moved to a different repository: https://github.com/moolitayer/awx-client-go. We will need to do these changes there, make a new release, and then update this project to use that new release.

Reply:
Done. https://github.com/moolitayer/awx-client-go/pull/5